plugins/GSdx/GSdx.rc

```diff
@@ -236,6 +236,8 @@ BEGIN
     COMBOBOX        IDC_ADAPTER,71,55,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     LTEXT           "Renderer:",IDC_STATIC,6,72,34,8
     COMBOBOX        IDC_RENDERER,71,70,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    COMBOBOX        IDC_MIPMAP,71,100,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    LTEXT           "Mipmap (Insert):",IDC_MIPMAP_TEXT,6,102,53,8
```

Comment:
> And mipmapping is much better than just mipmap.

Yeah, I also like ``Mipmapping`` over ``Mipmap`` but I used the latter for consistency on Linux-Windows GUI.

@gregory38 

Any opinion on ``Mipmapping`` **VS** ``Mipmap`` and ``(Ins)`` **VS** ``(Insert)``?